src/main/java/org/cbioportal/application/file/export/mappers/CancerStudyMetadataMapper.java

```diff
@@ -0,0 +1,11 @@
+package org.cbioportal.application.file.export.mappers;
+
+import java.util.List;
+import org.cbioportal.application.file.model.CancerStudyMetadata;
+import org.cbioportal.application.file.model.CancerType;
+
+public interface CancerStudyMetadataMapper {
```

Comment:
Can we add some javadocs to this. So that we can explicitly say why we have different mappers here.

Reply:
For all the mappers